src/main/java/org/spongepowered/common/mixin/core/world/gen/MixinChunkProviderServer.java

```diff
@@ -116,6 +116,15 @@ public ChunkDataStream getGeneratedChunks() {
         return WorldStorageUtil.doesChunkExist(this.world, this.chunkLoader, chunkCoords);
     }
 
+    // This method exists to allow us to speed up world pre generation.
+    @Override
+    public boolean doesChunkExistSync(Vector3i location) {
```

Comment:
It isn't used by the code you linked, it calls `chunkExists`, not `doesChunkExistSync `. It actually looks like the implementation of this method is copied into the code you linked.